### PR TITLE
Buy pack autoconfirmation

### PR DESCRIPTION
**Pull request by kielon**
Added autoconfirmation for buy pack dialog

### CODE REVIEW

**Comment by renard0** [comment]
Its completely the same as buy now confirm script? 
Have sense to merge them.
@Mardaneus86 what do you think?

File: confirm-buypack-dialog.user.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    var obsConfig           = { childList: true, characterData: true, attributes: false, subtree: true };
+  
+    targetNodes.each ( function () {
+        myObserver.observe (this, obsConfig);
+    } );
+  
+    function mutationHandler (mutationRecords) {
+      mutationRecords.forEach ( function (mutation) {
+        if ($(mutation.addedNodes).hasClass('Dialog')) {
+          if (gPopupClickShield._activePopup._title === "store.confirmation.questionTitle") {
```
Comment (renard0) [inline confirm-buypack-dialog.user.js]: Title value doesn't look unique.
Are you sure this test bypass only buy pack dialog?

**Comment by Mardaneus86** [comment]
This looks the same as the buy now confirmation script as @renard0 already said. I made a new issue to discuss a new idea I got because of this.

@kielon: please open a new issue before creating a completely new script, so we can discuss it beforehand. I want to prevent you do a lot of work that might not get merged in the end.

**Comment by Mit010** [comment]
Maybe it would also be possible to autoconfirm quicksell all items and send to transferlist for the doubles>

**Comment by Mardaneus86** [comment]
This PR is no longer suited for the new solution architecture and didn't add new functionality.

**Pull request closed without merging**